.gitignore

```diff
@@ -139,3 +140,6 @@ install_manifest.txt
 *.cmake
 *.cmake.in
 *.db
+
```

Comment:
optional/suggestion: Let's make a `temp` dir or something and add that to .gigitnore and any temporary files used for local testing only can be added to temp.